gio/gunixmounts.c

```diff
@@ -359,6 +359,27 @@ G_LOCK_DEFINE_STATIC(getmntent);
 #endif
 
 static GList *
+_get_mount_points_list_from_table (GHashTable *table)
+{
+  GList *result = NULL;
+  GHashTableIter iter;
+  gpointer key = NULL;
+  gpointer value = NULL;
+
+  if (table == NULL)
+    return NULL;
+
+  g_hash_table_iter_init (&iter, table);
+  while (g_hash_table_iter_next (&iter, &key, &value))
+    {
+      if (G_LIKELY (value != NULL))
```

Comment:
I don't really understand how the value can be `NULL` here. If it can't, you can just use `g_hash_table_get_values()`